dataland-e2etests/build.gradle.kts

```diff
@@ -1,3 +1,5 @@
+//dataland-e2etest

```

Comment:
ist der kommentar notwendig?

Reply:
Ich glaube es ist hilfreich um sich in der IDE zurechtzufinden.